README.md

```diff
@@ -98,6 +98,16 @@ include_directories(${SDL2_DIR}/include)
 target_link_libraries(${PROJECT_NAME} ${SDL2_LIB_DIR}/libSDL2.dll.a ${SDL2_LIB_DIR}/libSDL2main.a -mwindows)
 ```
 
+#### voll-statisch linken
+
+Damit die erzeugten .exe files auch portabel sind, *sollte* man voll-statisch bauen, d.h. alle benötigten Libraries werden in die fertige .exe gepackt und müssen auf anderen Rechnen **nicht** im genau gleihen Ordner vorhanden sein.
```

Comment:
Kleiner Rechtschreibfehler in dieser Zeile: *"gleihen" Ordner*

Reply:
danke